modules/sc-mesh-secure-deployment/src/nats/mdm_agent.py

```diff
@@ -880,12 +898,19 @@ def signal_handler(signum, frame):
     parser.add_argument("-k", "--keyfile", help="TLS keyfile", required=False)
     parser.add_argument("-c", "--certfile", help="TLS certfile", required=False)
     parser.add_argument("-r", "--ca", help="ca certfile", required=False)
-    parser.add_argument("-i", "--interface", help="e.g. br-lan or bat0", required=False)
+    parser.add_argument(
+        "-i",
+        "--interface",
+        help="Interface from where to monitor MDM server. Default is bat0",
```

Comment:
You can add a `default=Constants.LOWER_BATMAN.value` instead of manually doing it in an `if` below :waffle:

Reply:
Fixed in commit 1826923e3d821a8ab2c2b156ce5c2d2db02a0359.